api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -35,9 +35,12 @@
 	private String description;
 	
 	private Set<Integer> memberIds;
```

Comment:
You need to remove this field, deprecate its getter and setter. You will also need to make sure getMembers() loops through the set of CohortMembers and returns a set of their patientIds.  On the other hand setMembers() also would need to be updated to create instances of CohortMember and defaulting to current date date as startDate. 
I added comment on the ticket to get ideas on what should happen when removeMember() is called